src/components/MultiSelect/multiSelectChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Bug Fix",
+    affects: ["Functionality"],
+    notes: [
+      "Fixes bug where clicking a checkbox closes the component when `closeOnBlur` is true.",
+    ],
+  },
```

Comment:
Sorry, still not sure how to handle the scenario where there is two updates in version "Prerelease" with two different types, since type cannot be an array like affects can. @bigfishdesign13 @EdwinGuzman

Reply:
Since the intial changes for `closeOnBlur` have not yet been released, I don't think we need to acknowledge this update.  You can remove this entry.  

However, that does not address the issue of mixed update `type`s.  In the future. I think each `type` value can have a separate changelog entry.  If they require different type values, then the changes differ enough to warrant that distinction and separation.